pyrsa/data/dataset.py

```diff
@@ -299,6 +299,286 @@ def sort_by(self, by):
         order = np.argsort(desc)
         self.measurements = self.measurements[order]
         self.obs_descriptors = subset_descriptor(self.obs_descriptors, order)
+        
+
+class DatasetTime(Dataset):
```

Comment:
So this is a minor, aesthetical thing but I tend to prefer ChildParent class naming,  e.g. DogMammal instead of MammalDog, so in this case perhaps `TimeDataset` or `TemporalDataset`? What do you think?